test/pubtator/pubtator.js

```diff
@@ -12,14 +16,70 @@ import { pubtator_7 } from './10.15252_embj.2023113616.json';
  * annotations array of objects where each object indicates the id and the type of the bioentity (i.e gene, species, etc).
 */
 const bioCDocuments = [
-    //0..1
+    //0..1 (one organism hint in abstract)
     pubtator_2, // pmid: 31003867 | 0 organism hints in title { } | 1 organism hints in abstract: { 10090 aka (Mus musculus): 1 } | text: [ [] , ["mouse"] ]
-    //0..0
-    pubtator_4, // pmid: 38309274 | 0 organism hints in title { } | 0 organism hints in abstract: { } | text: [ [] , [] ]
-    //0..1
+    //0..1 (one organism hint in abstract)
     pubtator_5, // pmid: 33664495 | 0 organism hints in title { } | 1 organism hints in abstract: { 10090 aka (Mus musculus): 1 } | text: [ [] , ["mice"] ]
-    //1..*
+    //1..* (one organism hint in title and multiple organism hints in abstract)
```

Comment:
These map tests are pretty general, which means they won't assess how these 'hints' will be aggregated.